myning/chapters/enter_time_machine.py

```diff
@@ -66,8 +66,8 @@ def play():
 
         if option == "View Potential Macguffin":
             value = get_total_value()
-            xp_boost_str = str(xp_boost * 100) + "%"
-            mineral_boost_str = str(mineral_boost * 100) + "%"
+            xp_boost_str = str(round(xp_boost * 100, 2)) + "%"
+            mineral_boost_str = str(round(mineral_boost * 100, 2)) + "%"
```

Comment:
```suggestion
            xp_boost_str = f"{round(xp_boost * 100, 2)}%"
            mineral_boost_str = f"{round(mineral_boost * 100, 2)}%"
```
since we're here